franck-bereket.html

```diff
@@ -0,0 +1,11 @@
+<main id="main">
+    <link rel="stylesheet" href="styles.css">
+    <title id="title">Dr Norman</title>
```

Comment:
I think it would be better to include these lines in a head tag because the standard for writing an HTML file requires the presence of the title tag in the head tag, as well as for the link tag